README.md

```diff
@@ -28,31 +28,32 @@ selection of targets to use for communication with SRV-located services.
 It presents this service in the following interface:
```

Comment:
GH won't let me put this on the relevant line number above, but random thought I had while re-reading - the common case by far is to have SRV records that all use the same port on different hosts. Obviously records for different ports on the same host are valid too, but I'd make the example look more like the common case (and use 443 or 80 for the port so that a familiar port number stands out from the random numbers).

Reply:
Changed them to port 443 on `test1.example.com`, `test2.example.com`, `test3.example.com`